changelog/unreleased/bugfix-topbar-focus-management

```diff
@@ -0,0 +1,6 @@
+Bugfix: Focus management in topbar
+
+We've fixed issues with focus management upon opening and closing 
+the dropdown menus in the ApplicationSwitcher and Usermenu component.
```

Comment:
Changelogs are not necessarily dev facing. `component` is probably not something that everyone understsands. 😁  You can probably just leave out the word `component`, the sentence is already self explanatory without it.